docs/version_3.md

```diff
@@ -0,0 +1,136 @@
+# Version 3.0!
+
+We have been working hard the past few months on a number of features and improvements we wanted to combine into a major release. Our aim is that is next version is backwards compatible, we thought it was a good idea to move to a new major version because we are changing a core part of how Style Dictionary works. 
+
+If you would like to get a pre-release of 3.0, use the `next` tag to install Style Dictionary:
+
+`npm install --save-dev style-dictionary@next`
+
+If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!
+
+### Transitive transforms
+
+This is the big one that required a big re-architecture of how the Style Dictionary build process works. 
+
+Up until now the build process looked like this: https://amzn.github.io/style-dictionary/#/build_process
+After merging all of the token files, it would go through the merged object and transform tokens it found, but only do value transforms on tokens that did not reference another token. The original intent here was that if you have a color for example, the value of it should be the same for all references of it, so we only need to do a value transform once. Then after all tokens are transformed, resolve all aliases/references. 
+
+However, we heard from the community there were a number of reasons why someone might want to transform the value of an aliased token. 
+
+* https://github.com/amzn/style-dictionary/issues/451
+* https://github.com/amzn/style-dictionary/issues/452
+* https://github.com/amzn/style-dictionary/issues/208
+
+The new build process is similar, except that it does multiple passes of transforming and resolving aliases and defers transforms if the token has an unresolved alias. Each pass might reveal tokens that are ready to be transformed in the next pass. Take this example:
```

Comment:
The new build process is similar, except that it does multiple passes of transforming and resolving aliases and defers transforms if the token has an unresolved alias. -> The new build process is similar, except that it recursively transforms and resolves aliases, only deferring a transform to the next cycle if the token has an unresolved alias.

(Lets be explicit. I think this is accurate. If not, lets correct it.)